packages/account/src/Sections/Security/ClosingAccount/__tests__/closing-account-has-pending-conditions.spec.tsx

```diff
@@ -0,0 +1,253 @@
+                    is_virtual: 0,
+                    icon: 'usd',
+                    title: 'USD',
+                },
+                {
+                    loginid: 'CR234',
+                    is_virtual: 0,
+                    icon: 'btc',
+                    title: 'BTC',
+                },
+                {
+                    loginid: 'VRTC90000148',
+                    is_virtual: 1,
+                    icon: 'real',
+                    title: 'Real',
+                },
+            ],
+
+            is_eu: false,
+        },
+    });
+
+    it('should render the ClosingAccountHasPendingConditions component', () => {
+        const { container } = render(
+            <StoreProvider store={store}>
+                <ClosingAccountHasPendingConditions {...mock_props} />
+            </StoreProvider>
+        );
+        expect(container).toBeInTheDocument();
+    });
+
+    it('should show the accounts with deriv_balance', () => {
+        const details = {
+            balance: {
+                CR123: {
+                    balance: 2744.0,
+                    currency: 'USD',
+                },
+                CR234: {
+                    balance: 0.18833116,
+                    currency: 'BTC',
+                },
+            },
+        };
+
+        render(
+            <StoreProvider store={store}>
+                <ClosingAccountHasPendingConditions {...mock_props} details={details} />
+            </StoreProvider>
+        );
+        expect(
+            screen.getByText(/please withdraw your funds from the following deriv account\(s\):/i)
+        ).toBeInTheDocument();
+        expect(screen.getByText(/2,744\.00/i)).toBeInTheDocument();
+        expect(screen.getByText(/USD/i)).toBeInTheDocument();
+
+        expect(screen.getByText(/0\.18833116/i)).toBeInTheDocument();
+        expect(screen.getByText(/BTC/i)).toBeInTheDocument();
+    });
+
+    it('should show the accounts with open_positions', () => {
+        const details = {
+            open_positions: {
+                CR123: 1,
+            },
+        };
+
+        render(
+            <StoreProvider store={store}>
+                <ClosingAccountHasPendingConditions {...mock_props} details={details} />
+            </StoreProvider>
+        );
+        expect(
+            screen.getByText(/please close your positions in the following deriv account\(s\):/i)
+        ).toBeInTheDocument();
+        expect(screen.getByText(/1 position/i)).toBeInTheDocument();
+    });
+
+    it('should show the accounts with pending withdrawals', () => {
+        const details = {
+            pending_withdrawals: {
+                CR123: 1,
+            },
+        };
+
+        render(
+            <StoreProvider store={store}>
+                <ClosingAccountHasPendingConditions {...mock_props} details={details} />
+            </StoreProvider>
+        );
+        expect(screen.getByText(/Pending withdrawal request:/i)).toBeInTheDocument();
```

Comment:
lets remove case insensitive regex. QA usually checks if the text are title cased etc. as per figma design.

Reply:
case insensitive is usually preferred approach for test cases

Reply:
oh, ok.. 👌🏼